src/components/navigations/app-bar/app-bar.component.js

```diff
@@ -6,15 +6,48 @@ import IconButton from '@material-ui/core/IconButton';
 import Typography from '@material-ui/core/Typography';
 import Badge from '@material-ui/core/Badge';
 import Hidden from '@material-ui/core/Hidden';
+import Menu from '@material-ui/core/Menu';
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
I would put this at the top of the material imports.

Reply:
Has been removed